source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -360,6 +360,7 @@ export class StringPropertyEditor extends BaseStringPropertyEditor {
                     onExecute: (sender: SingleInputPropertyEditor, clickedElement: HTMLElement) => {
                         clickedElement.setAttribute("aria-expanded", "true");
 
+                        // Question: why does FieldPicker need dataStructure?
                         let fieldPicker = new FieldPicker(context.designContext.dataStructure);
```

Comment:
Does anyone know why we do this?

Reply:
I believe this is how the data binding picker is powered (and how it generates binding expressions):

sample data payload:

```json
{
    "test": [
        "boring",
        "strings"
    ]
}
```

![image](https://user-images.githubusercontent.com/16614499/180380925-0ef9afc4-ca40-4136-8159-8ff3e9547401.png)

clicking on `test` yields the binding expression `${$root['test[0]']}`, which... isn't really what I wanted, but I think that's what's at play here...

Reply:
Okay that makes sense. Do we want to be able to access `$host` parameters from the `Bind...` options as well?

@paulcam206

Reply:
After looking into this, I think it'd be better to complete this when its higher priority. I created an issue on github to track it :)

https://github.com/microsoft/AdaptiveCards/issues/7724